src/api/eyeAccount.js

```diff
@@ -0,0 +1,34 @@
+import client, {createEntity, deleteEntity, updateEntity} from "./client";
+
+/**
+ * The root API endpoint 
+ */
+const EYE_ACCOUNT_ENDPOINT = '/eyeAccountList'
+
+export const getEyeAccountFetcher = () => {
+  return [EYE_ACCOUNT_ENDPOINT, async () => { 
+    const response = await client.get(EYE_ACCOUNT_ENDPOINT)
+    return response.data
+  }]
+}
+
+/**
+ * Creates 
+ */
+export const createEyeAccount = async ({headmateId, name, spaceTitle, changeTheme, headingFont, headingFontSize, mainFont, mainFontSize, createPassword, password, connectImg, entry}) => {
```

Comment:
It might be worth reformatting this at some stage given the number of fields makes things fairly hard to read.